src/cryptoadvance/specter/templates/device/new_device/new_device_mnemonic.jinja

```diff
@@ -208,5 +303,66 @@
                 advancedButton.innerHTML = `{{ _("Advanced") }} &#9660;`;
             }
         }
+
+
+        let scanner = document.getElementById('seedqr-scan');
+        if (scanner != null) {
+            scanner.addEventListener('scan', async (e) => {
+                let result = e.detail.result;
+                if(result==null){
+                    return;
+                }
+
+                if (result.startsWith("StandardSeedQR:")) {
+                    let digitstream = result.substring("StandardSeedQR:".length);
+                    parseSeedQR(digitstream, null);
+
+                } else if (result.startsWith("CompactSeedQR:")) {
+                    let bytestream = result.substring("CompactSeedQR:".length);
+                    parseSeedQR(null, bytestream);
+                }
```

Comment:
Error feedback is missing here. I would at least add an else clause showing an error like "Sth. went wrong", perhaps with a reminder that compact SeedQR is not supported yet.